src/CommaSeparatedJsonParser.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+class CommaSeparatedJsonParser extends AbstractValueParser
+{
+    protected function parse($value)
+    {
+        if (!empty($value) && $value[0] !== '[' && $value[strlen($value) - 1] !== ']') {
```

Comment:
For input `[{"a":1}]` (a json array, no comma as there is only a single element in the input string) the expected output would be `[ [ [ "a" => 1 ] ] ]` but is `[ [ "a" => 1 ] ]` (caused by the `&& $value[0] !== '[' && $value[strlen($value) - 1] !== ']'`)